### PR TITLE
Fix style btn filters

### PR DESCRIPTION
**Pull request by massimez**

### CODE REVIEW

File: components/app/panels/filters.vue [hunk 1]
```diff
@@ -177,12 +177,18 @@ export default {
...
+  }
 
 }
 
 .item-btn {
   padding: 0 20px;
-
+  border: 1px solid transparent;
+  &:hover{
+    //border: 1px solid $black100;
```
Comment (matykoks) [inline components/app/panels/filters.vue:190]: ?

File: assets/scss/vars.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 $black0: #F7F8FA;
-$black100: #E6EAEE;
+$black100: #E9EDF2;
```
Comment (matykoks) [inline assets/scss/vars.scss:2]: ?

Comment (massimez) [reply assets/scss/vars.scss:2]: From Figma

File: components/app/panels/filters.vue (hunk 1 above)
Comment (massimez) [reply components/app/panels/filters.vue]: Ive removed it

**Pull request merged**